terraform/datadog/elb.tf

```diff
@@ -0,0 +1,67 @@
+resource "datadog_monitor" "abnormal_api_latency_cc" {
+  name    = "${format("%s Abnormal API Latency - CC", var.env)}"
+  type    = "query alert"
+  message = "${format("{{#is_alert}}We're experiencing >= {{threshold}} change in ELB Latency.{{/is_alert}} \n{{#is_warning}}We're experiencing >= {{warn_threshold}} change in ELB Latency.{{/is_warning}} \n\nVisit the [Team Manual > Responding to alerts > API Latency](%s#api-latency) for more info.", var.datadog_documentation_url)}"
+
+  query = "${format("avg(last_15m):anomalies(avg:aws.elb.latency{name:%s-cf-cc}, 'basic', 2, direction='both') > 0.3", var.env)}"
+
+  thresholds {
+    warning  = 0.15
```

Comment:
I'm assuming we've not extracted these to variables because we think they're accidentally the same across all monitors? ie we expect that the cc monitor may have different values to the uaa monitor?

Reply:
Also is the plan to deploy it, get some feedback around typical behaviour and then adjust the values?

Reply:
Yes, we anticipate that each monitor might need different config, and thought it'd be premature to extract these into variables at this time.